mozilla/fx-private-relay
# MPP-2343

This PR fixes MPP-2343
 
# New feature description

On the “Verify your true phone number” page, if a user types a phone number starting with 1 the input field’s formatting ends up broken.

# Screenshot (if applicable)

<img width="674" alt="image" src="https://user-images.githubusercontent.com/3924990/189550240-66aa3f17-ecc3-4a22-9db5-046cd409587d.png">

# How to test

Go to the phone page, without having verified a real phone number. Try typing a number that starts with 1, it should not break the input format. The input component should let you type a zip code and the 7-digit number without issue. 

# Checklist
